R/cppproperties/extractLinkingTo.R

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Does it make sense to use the same name of `cppProperties` in the folder name to be consistent with the ts file?

Reply:
Agreed b639e55